advanced/advanced_numpy/examples/wavreader.py

```diff
@@ -4,6 +4,7 @@
 
 An exercise to learn complex dtypes.
 """
+from __future__ import print_function
```

Comment:
Same remark: "print(a)" works in Python 2, so we don't need this line in this example.